### DISCUSSION TITLE
kotatogram setting will not change to Chinese font

### DISCUSSION THREAD

**Discussion by YasuoQAQ** [kotatogram/kotatogram-desktop#375] [EN 2. Q&A]
Please add Chinese language in the software

**Comment by EricKotato** [comment]
So, the language or the font? You can use system font (Kotatogram Settings > Change application fonts > select a font). If you mean language, then there are already some languages that you can used even in TDesktop:

- https://t.me/setlanguage/zh-hans-beta
- https://t.me/setlanguage/zh-hant-beta

Some strings may be not translated in Kotatogram, so if you're willing to help with translation, you're welcome to Kotatogram's Crowdin page: https://crowdin.com/project/kotatogram-desktop
